### PR TITLE
use rpmautospec

### PR DESCRIPTION
**Pull request by TomasTomecek**
https://docs.pagure.org/Fedora-Infra.rpmautospec/principle.html

### CODE REVIEW

File: fedora/changelog [hunk 1]
```diff
@@ -1,53 +1,3 @@
...
-%files -n python3-%{srcname}
-%license LICENSE
-%doc README.md
-%{python3_sitelib}/%{srcname}
-%{python3_sitelib}/%{srcname}-%{version}-py?.?.egg-info
-
-
-%changelog
 * Wed Apr 01 2020 Jan Sakalos <sakalosj@gmail.com> - 0.11.1-1
 - patch release: 0.11.1
```
Comment (lachmanfrantisek) [inline fedora/changelog:2]: Do we need this file? Cannot it use only git log?

**Comment by lachmanfrantisek** [comment]
recheck

File: fedora/changelog (hunk 1 above)
Comment (TomasTomecek) [reply fedora/changelog:2]: rpmautospec doesn't process git log entriess past the latest build.

You are asking here to drop all the changelog entries which is fine to do in upstream, but terrible to do in Fedora. Hence we could not include the file here and have it downstream only, which makes sense to me.

**Comment by TomasTomecek** [comment]
> Looks good. You probably want to release soon, don't you?

no need to, I'm fine to wait.

File: fedora/changelog (hunk 1 above)
Comment (lachmanfrantisek) [reply fedora/changelog:2]: OK, I get it now. I misunderstood that part from their docs.

File: fedora/python-ogr.spec [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+%package -n     python3-%{srcname}
+Summary:        %{summary}
+%{?python_provide:%python_provide python3-%{srcname}}
+
+%description -n python3-%{srcname}
+One Git library to Rule!
+
+
+%prep
+%autosetup -n ogr-0.11.1
```
Comment (jpopelka) [inline fedora/python-ogr.spec:30]: Sorry for being late for the party, but why can't we use `%{srcname}-%{version}`?
I don't see it mentioned in https://docs.pagure.org/Fedora-Infra.rpmautospec/principle.html

Comment (TomasTomecek) [reply fedora/python-ogr.spec:30]: because I'm an ignorant moron, obviously https://github.com/packit-service/ogr/pull/388

**Pull request merged**